firmware_configurations/klipper/TechnoPhreak/BTT_SKR_v2/printer.cfg

```diff
@@ -0,0 +1,608 @@
+stealthchop_threshold: 0
+
+##	Z MCU - In Y Position
+##	Z1 Stepper - Rear Left
+[stepper_z1]
+step_pin: z:PD5
+dir_pin: z:PD4
+enable_pin: !z:PD6
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z1]
+uart_pin: z:PD3
+interpolate: true
+run_current: 0.8
+hold_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+##	Z MCU - In Z Position
+##	Z2 Stepper - Rear Right
+[stepper_z2]
+step_pin: z:PA15
+dir_pin: !z:PA8
+enable_pin: !z:PD1
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z2]
+uart_pin: z:PD0
+interpolate: true
+run_current: 0.8
+hold_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+##	Z MCU - In E0 Position
+##	Z3 Stepper - Front Right
+[stepper_z3]
+step_pin: z:PD15
+dir_pin: z:PD14
+enable_pin: !z:PC7
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 stepper_z3]
+uart_pin: z:PC6
+interpolate: true
+run_current: 0.8
+hold_current: 0.8
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+
+#####################################################################
+# 	Extruder
+#####################################################################
+
+#	E0 on MCU X/Y
+[extruder]
+step_pin: PD15
+dir_pin: !PD14
+enable_pin: !PC7
+##	Update value below when you perform extruder calibration
+##	If you ask for 100mm of filament, but in reality it is 98mm:
+##	rotation_distance = <previous_rotation_distance> * <actual_extrude_distance> / 100
+##  22.6789511 is a good starting point
+rotation_distance: 22.6789511	#Bondtech 5mm Drive Gears
+##	Update Gear Ratio depending on your Extruder Type
+##	Use 50:17 for Afterburner/Clockwork (BMG Gear Ratio)
+##	Use 80:20 for M4, M3.1
+gear_ratio: 50:17				#BMG Gear Ratio
+microsteps: 16
+full_steps_per_rotation: 200	#200 for 1.8 degree, 400 for 0.9 degree
+nozzle_diameter: 0.400
+filament_diameter: 1.75
+heater_pin: PB3
+##	Validate the following thermistor type to make sure it is correct
+sensor_type: ATC Semitec 104GT-2
+sensor_pin: PA2
+min_temp: 10
+max_temp: 270
+max_power: 1.0
+min_extrude_temp: 170
+control = pid
+pid_kp = 26.213
+pid_ki = 1.304
+pid_kd = 131.721
+##	Try to keep pressure_advance below 1.0
+#pressure_advance: 0.05
+##	Default is 0.040, leave stock
+#pressure_advance_smooth_time: 0.040
+
+##	E0 on MCU X/Y
+##	Make sure to update below for your relevant driver (2208 or 2209)
+[tmc2209 extruder]
+uart_pin: PC6
+interpolate: false
+run_current: 0.5
+hold_current: 0.4
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+#####################################################################
+# 	Bed Heater
+#####################################################################
+
+[heater_bed]
+##	SSR Control - Z board, BED Pin
+heater_pin: z:PB7
+sensor_type: NTC 100K beta 3950

```

Comment:
change this to `Generic 3950` as the current setting is deprecated. also comment out like the extruder sensor.

Reply:
Should be good now